apps/next/next.config.ts

```diff
@@ -21,6 +21,22 @@ const nextConfig: NextConfig = {
   transpilePackages: ['@accelint/design-foundation', '@accelint/design-system'],
   productionBrowserSourceMaps: true,
 
+  /**
+   * ⚠️ TURBOPACK NOT SUPPORTED
```

Comment:
just a note: hopefully will be supported in the near future: https://github.com/gohypergiant/standard-toolkit/pull/650